targets/targets.json

```diff
@@ -622,7 +622,7 @@
     },
     "EV_COG_AD4050LZ": {
         "inherits": ["Target"],
-        "core": "Cortex-M4",
+        "core": "Cortex-M4F",
```

Comment:
This does not seem to be related to armc6, shall be separate commit

Reply:
Cortex-M4 for ARMC6 result in all SYNC_FAILED in all greentea tests.
Cortex-M4F for ARMC5 and ARMC6 pass for greentea tests.

Reply:
Cortex-M4 is also an issue to ADuCM4050 IAR exporter because Embedded Workbench is pre-configured by ADuCM4050 Board Support Package to assume that ADuCM4050's FPU is present and enabled for IDE. The inconsistent FPU settings in IAR IDE will cause Bus fault Exception eventually.